rpcs3/Crypto/unpkg.h

```diff
@@ -333,7 +341,8 @@ class package_reader
 	std::string m_path{};
 	std::string m_install_dir{};
 	fs::file m_file{};
-	std::unique_ptr<u128[]> m_buf{};
+	fs::file m_memory_file;
+	std::vector<std::unique_ptr<u128[]>> m_bufs{};
```

Comment:
Why not just use a std::vector<u128> instead of this pointer to c-style array

Reply:
It's fine here, it's the closest we have to a vector with constant size. It is also a bit faster to dereference because it's only 8 bytes instead of a vector which is 24 so it uses built-in addressing.